homeassistant/components/motioneye/config_flow.py

```diff
@@ -82,7 +83,9 @@ def _get_form(
             )
 
         if user_input is None:
-            return _get_form(reauth_entry.data if reauth_entry else {})
+            return _get_form(
+                cast(Dict[str, Any], reauth_entry.data) if reauth_entry else {}
```

Comment:
Will this work? Should be a little cleaner.
```suggestion
                cast(ConfigType, reauth_entry.data) if reauth_entry else {}
```